10:45:51 -     INFO - wally           - All info would be stored into /home/mosorio/wally/docker_results/intent_bass
10:45:51 -     INFO - wally           - Start discover_stage
10:45:51 -     INFO - wally           - Start reuse_vms_stage
10:45:51 -     INFO - wally           - Start log_nodes_statistic_stage
10:45:51 -     INFO - wally           - Found 2 nodes total
10:45:51 -    DEBUG - wally           - Found 2 nodes with role testnode
10:45:51 -     INFO - wally           - Start save_nodes_stage
10:45:51 -     INFO - wally           - Start connect_stage
10:45:51 -     INFO - wally           - Connecting to nodes
10:45:52 -     INFO - wally           - All nodes connected successfully
10:45:52 -     INFO - wally           - Start run_tests_stage
10:45:52 -    DEBUG - wally           - Run preparation
10:45:52 -     INFO - wally           - Prefilling file /mnt/test.raw
10:45:52 -    DEBUG - wally           - SSH:170.80.74.57:22 Exec 'sudo fio --name=xxx --filename=/mnt/test.raw --direct=1 --bs=4m --size=20480m --rw=write'
10:45:52 -     INFO - wally           - Prefilling file /mnt/test.raw
10:45:52 -    DEBUG - wally           - SSH:138.219.231.52:22 Exec 'sudo fio --name=xxx --filename=/mnt/test.raw --direct=1 --bs=4m --size=20480m --rw=write'
10:46:55 -     INFO - wally           - Initiall fio fill bw is 324 MiBps for this vm
10:47:27 -    ERROR - wally           - XXXX
Traceback (most recent call last):
  File "wally/suits/io/fio.py", line 669, in pre_run_th
    self.prefill_test_files(rossh, files, force)
  File "wally/suits/io/fio.py", line 554, in prefill_test_files
    rossh(cmd, timeout=curr_sz)
  File "wally/suits/itest.py", line 204, in closure
    **kwargs)
  File "wally/ssh_utils.py", line 543, in run_over_ssh
    raise OSError(templ.format(node, cmd, code, output))
OSError: SSH:138.219.231.52:22 Cmd 'sudo fio --name=xxx --filename=/mnt/test.raw --direct=1 --bs=4m --size=20480m --rw=write' failed with code 128. Output: xxx: (g=0): rw=write, bs=4M-4M/4M-4M/4M-4M, ioengine=sync, iodepth=1
fio-2.2.8
Starting 1 process
xxx: Laying out IO file(s) (1 file(s) / 20480MB)

fio: terminating on signal 15

fio: terminating on signal 15
xxx: Laying out IO file(s) (1 file(s) / 20480MB)

fio: terminating on signal 15

xxx: (groupid=0, jobs=1): err= 0: pid=31655: Thu Feb  9 13:47:01 2017
  write: io=5584.0MB, bw=60890KB/s, iops=14, runt= 93908msec
    clat (msec): min=3, max=1216, avg=67.02, stdev=178.58
     lat (msec): min=3, max=1216, avg=67.26, stdev=178.58
    clat percentiles (msec):
     |  1.00th=[    4],  5.00th=[    5], 10.00th=[    5], 20.00th=[    6],
     | 30.00th=[    8], 40.00th=[   15], 50.00th=[   22], 60.00th=[   26],
     | 70.00th=[   27], 80.00th=[   29], 90.00th=[  122], 95.00th=[  375],
     | 99.00th=[ 1029], 99.50th=[ 1037], 99.90th=[ 1123], 99.95th=[ 1221],
     | 99.99th=[ 1221]
    bw (KB  /s): min= 3976, max=422603, per=100.00%, avg=76043.52, stdev=88487.71
    lat (msec) : 4=3.01%, 10=31.45%, 20=13.25%, 50=39.33%, 100=2.15%
    lat (msec) : 250=4.30%, 500=2.44%, 750=1.43%, 1000=1.00%, 2000=1.65%
  cpu          : usr=0.39%, sys=0.69%, ctx=1398, majf=0, minf=55
  IO depths    : 1=100.0%, 2=0.0%, 4=0.0%, 8=0.0%, 16=0.0%, 32=0.0%, >=64=0.0%
     submit    : 0=0.0%, 4=100.0%, 8=0.0%, 16=0.0%, 32=0.0%, 64=0.0%, >=64=0.0%
     complete  : 0=0.0%, 4=100.0%, 8=0.0%, 16=0.0%, 32=0.0%, 64=0.0%, >=64=0.0%
     issued    : total=r=0/w=1396/d=0, short=r=0/w=0/d=0, drop=r=0/w=0/d=0
     latency   : target=0, window=0, percentile=100.00%, depth=1

Run status group 0 (all jobs):
  WRITE: io=5584.0MB, aggrb=60889KB/s, minb=60889KB/s, maxb=60889KB/s, mint=93908msec, maxt=93908msec

Disk stats (read/write):
  sdb: ios=0/12574, merge=0/6, ticks=0/227047, in_queue=228536, util=99.32%

